chefignore

```diff
@@ -93,9 +100,3 @@ Strainerfile
 ###########
 .vagrant
 Vagrantfile
-
-# Travis #
-##########
-.travis.yml
-
-.kitchen
```

Comment:
we want to keep .kitchen here otherwise it gets included in the berks install and if you use Hyper-V (like me) that includes the test image file.